packages/integration-tests/src/entities.ts

```diff
@@ -75,3 +75,5 @@ export enum BlockExplorer {
   baseUrl = "http://localhost:3010",
   localNetwork = "/?network=local",
 }
+
+export const txSumEth = "0.000009";
```

Comment:
const txSumETH = '0.000009';
export const Values = {
  txSumETH: txSumETH,
};

Reply:
May be it will be more flexible in case we would like to extend it later